projects/core/components/primitive-textfield/primitive-textfield.template.html

```diff
@@ -87,13 +87,18 @@
                 <tui-svg class="custom-icon" [src]="iconSrc"></tui-svg>
             </ng-template>
         </div>
-        <tui-svg
+        <span
```

Comment:
I personally prefer `span` when flex or other styles are applied because `div` is invalid content for some containers whereas `span` can be put almost everywhere and layout would be valid.